roles/ceph-nfs/tasks/pre_requisite_non_container.yml

```diff
@@ -41,12 +41,27 @@
   apt_repository:
     repo: "deb {{ nfs_ganesha_stable_deb_repo }} {{ ceph_stable_distro_source | default(ansible_lsb.codename) }} main"
     state: present
-  changed_when: false
+    update_cache: no
+  register: add_ganesha_apt_repo
+  when:
+    - ansible_os_family == 'Debian'
+    - nfs_ganesha_stable
+    - ceph_origin == 'repository'
+    - ceph_repository == 'community'
+
+- name: update apt cache
+  apt:
+    update_cache: yes
+  register: update_ganehsa_apt_cache
```

Comment:
spelling error here, resulting in the registered var and the used var not matching

Reply:
👍 fixed! thanks